src/content/cookbook/architecture/command.md

```diff
@@ -0,0 +1,616 @@
+---
+title: Command pattern
+description: Improve ViewModels with Commands
+js:
+  - defer: true
+    url: /assets/js/inject_dartpad.js
+---
+
+<?code-excerpt path-base="cookbook/architecture/command"?>
+
+Model-View-ViewModel (MVVM) is a design pattern 
+that separates a feature of an application into three parts: 
+the `Model`, the `ViewModel` and the `View`. 
+Views and ViewModels make up the UI layer of an application. 
+Repositories and services represent the data layer of an application, 
+or the Model layer of MVVM.
+
+ViewModels can become very complex 
+as an application grows 
+and features become bigger.
+The command pattern helps to streamline running actions in ViewModels 
+by encapsulating some of its complexity and avoiding code repetition.
```

Comment:
I will change it! I will also use https://docs.flutter.dev/app-architecture/case-study/ui-layer#command-objects as reference since it also defines what a command is.